tests/Application/Matching/AllocatorTest.php

```diff
@@ -310,4 +304,30 @@ public function testItRejectsFundingInWrongCurrency(): void
         // act
         $this->sut->allocateMatchFunds($donation);
     }
+
+    public function testReleaseMatchFundsSuccess(): void
```

Comment:
This is moved from the general donation repo test. Everything else came from the allocation-specific repo test, as git's picked up.